pathologies.php

```diff
@@ -26,11 +26,11 @@
                 echo pg_last_error($db);
             }
             else {
-                datatable(["ID", "Name"]);
+                datatable(["ID", "Nom", "Prénom","Adresse permanente","Adresse olympique","Téléphone"]);
 
                 while ($row = pg_fetch_row($ret)) {
                     echo "<tr>";
-                    for ($i = 0; $i < 2; $i++) {
+                    for ($i = 0; $i < 6; $i++) {
```

Comment:
I think you should limit it at 7, since you have 5 extra variables
for ($i = 0; $i < 7; $i++) { .......